### PR TITLE
Support Vanilla Hide&Seek & Fix Vents On Dleks Map

### PR DESCRIPTION
**Pull request by Tommy-XL**
1. Support Vanilla Hide&Seek
2. Fixed Vents On Dleks Map (For Modded)

### CODE REVIEW

File: Modules/TemplateManager.cs [hunk 1]
```diff
@@ -119,7 +134,11 @@ public static void SendTemplate(string str = "", byte playerId = 0xff, bool noEr
             playerName = () => Main.AllPlayerNames[playerId];   
         }
 
-        _replaceDictionary["PlayerName"] = playerName;
+        //if (GameStates.IsNormalGame)
+            _replaceDictionaryNormalOptions["PlayerName"] = playerName;
+        //else if (GameStates.IsHideNSeek)
+            _replaceDictionaryHideNSeekOptions["PlayerName"] = playerName;
+
```
Comment (Tommy-XL) [inline Modules/TemplateManager.cs:141]: For some reason, when creating a lobby, `"PlayerName"` is null
Need to find a solution to the problem

**Comment by Tommy-XL** [comment]
399 PR, Pog

File: Modules/TemplateManager.cs (hunk 1 above)
Comment (Tommy-XL) [reply Modules/TemplateManager.cs:141]: I found out that this only happens when creating a local lobby, I don’t know why, lol

**Comment by Tommy-XL** [comment]
I can say that at this stage H&S is at the testing stage

**Comment by Tommy-XL** [comment]
> What roles do we support in hide n seek?

I don't plan to support any roles in H&S at all, Just some settings that work well in H&S

**Comment by Tommy-XL** [comment]
like Decon time, Random maps, Disable ZipLine etc.

File: Modules/TemplateManager.cs (hunk 1 above)
Comment (Tommy-XL) [reply Modules/TemplateManager.cs]: Fixed

**Comment by NikoCat233** [comment]
> > like Decon time, Random maps, Disable ZipLine etc.
> 
> can you straight up disable decon, like make both doors permanently open

This isnt possible.
IS dont provide a flag to open both door

**Comment by 0xDrMoe** [comment]
ETA on this?

**Comment by Tommy-XL** [comment]
I think everything is ready

**Comment by Tommy-XL** [comment]
> I think everything is ready

Nope, lol

**Comment by Tommy-XL** [comment]
Now done
(Also in the future need to fix random spawn as the positions of the cards have been changed)

**Pull request merged**